bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,77 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
+    extract_documentation)
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class DocumentationStyleBear(LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/7sfk3i9oxs1ixg2ncsu3pym0u'
+    CAN_DETECT = {'Documentation'}
+
+    def run(self, filename, file, docstyle: str, language: str):
+        """
+        Checks for certain in-code documentation styles.
+
+        It currently checks for the following style:
+
+        ::
+
+            hello
+            :param x:
+                4 space indent
+            :return:
+                also 4 space indent
+                following lines are also 4 space indented
+        """
+        for doc_comment in extract_documentation(file, language, docstyle):
+            metadata = iter(doc_comment.parse())
+
+            # Assuming that the first element is always the only description.
+            new_metadata = [next(metadata)]
+
+            for m in metadata:
+                stripped_desc = list(map(str.lstrip, m.desc.splitlines(True)))
+                stripped_desc[0] = (('\n' + ' ' * 4 + stripped_desc[0])
```

Comment:
```
mak@hp-elitebook:~/dev/test$ cat test.py
def x:
 return 55
 
def qr:
    """
       this is heavy

    :param x:
    :return:"""return 88
    
def w:
    """ Hello world """
mak@hp-elitebook:~/dev/test$ cat .coafile
bears = DocumentationStyleBear
files = test.py
docstyle = default
language = python
default_actions = DocumentationStyleBear: ApplyPatchAction
bear_dirs = ../coala-bears/bears
mak@hp-elitebook:~/dev/test$ coala -L DEBUG
[DEBUG][18:36:19] Platform Linux -- Python 3.4.3, pip 8.1.2, coalib 0.8.0.dev99999999999999
[DEBUG][18:36:19] Since the configuration settings have changed since the last run, the cache will be flushed and rebuilt.
[DEBUG][18:36:19] The file cache was successfully flushed.
Executing section Default...
[DEBUG][18:36:19] coala is run only on changed files, bears' log messages from previous runs may not appear. You may use the `--flush-cache` flag to see them.
[DEBUG][18:36:19] Files that will be checked:
/home/mak/dev/test/test.py
[DEBUG][18:36:19] Running bear DocumentationStyleBear...
[WARNING][18:36:19] Bear DocumentationStyleBear failed to run. Take a look at debug messages (`-L DEBUG`) for further information.
[DEBUG][18:36:19] The bear DocumentationStyleBear raised an exception. If you are the writer of this bear, please make sure to catch all exceptions. If not and this error annoys you, you might want to get in contact with the writer of this bear.

Traceback information is provided below:

Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/coala-0.8.0.dev99999999999999-py3.4.egg/coalib/bears/Bear.py", line 250, in execute
    return [] if result is None else list(result)
  File "/home/mak/dev/coala-bears/bears/documentation/DocumentationStyleBear.py", line 45, in run
    if not stripped_desc[0] == ''
IndexError: list index out of range
mak@hp-elitebook:~/dev/test$ 
```